pipeline_options.md

```diff
@@ -101,6 +101,13 @@ may be of different lengths, and each job will be launched with a unique combina
 of array inputs. The manual adapter does not do any handling of outputs besides
 base delocalization.
 
+* `product`: If false (the default) all inputs must be of equal length or a constant.
+In this case, inputs will take one value per job. If true, arrays may be of unequal length
+and one job will be launched for each combination of elements
+* `arrays`: A list of input names which should be considered a bash array, instead of
```

Comment:
Strictly speaking, this isn't exported as a bash array, it's a set of tab-delimited strings.
We could add to the entrypoint something like
```bash
read -r -a [variable_name] <<< [input string] 
```

so that it truly is a bash array